src/AvailableCommandsPacket.php

```diff
@@ -43,21 +43,23 @@ class AvailableCommandsPacket extends DataPacket implements ClientboundPacket{
 	public const ARG_TYPE_WILDCARD_INT = 0x05;
 	public const ARG_TYPE_OPERATOR = 0x06;
 	public const ARG_TYPE_TARGET = 0x07;
-	public const ARG_TYPE_WILDCARD_TARGET = 0x08;
+	public const ARG_TYPE_WILDCARD_TARGET = 0x09;
 
 	public const ARG_TYPE_FILEPATH = 0x10;
 
-	public const ARG_TYPE_STRING = 0x20;
+	public const ARG_TYPE_POSITION = 0x20;
```

Comment:
From my analysis this appears incorrect. There is no symbol associated with `0x20`.

I see two possible values for position `0x2e` and `0x2f`. I don't know why there are 2 or what the difference between them is.

Reply:
`2e` appears to be `block position`, while `2f` appears to be any floating-point vector position.